### PR TITLE
fix(@desktop/community): Add tags row on communities portal page

### LINKED ISSUE #6734: Display tags in community portal
### Description

- [x] display tags
- [x] display `>`and support scrolling

181830825-b586e6a7-0704-46da-9d8e-1b744380e0fd

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #6734
Depends https://github.com/status-im/StatusQ/pull/844

Add row with tags. Only for display

### Affected areas

Communities
### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://user-images.githubusercontent.com/2522130/183500942-9b2c36bb-373d-4398-babe-8e9e3e259058.mov

### CODE REVIEW

**Comment by MishkaRogachev** [comment]
@noeliaSD do we have any rules and instructions how extract icons from figma?

**Comment by elina2015** [comment]
@MishkaRogachev could you kindly help with the builds for testing?

**Comment by noeliaSD** [comment]
> @noeliaSD do we have any rules/instructions how extract icons from figma?

AFAIK there is no specific rule to extract icons. I normally use the `export` option in figma or if needed, I ask directly to designers.

**Review by noeliaSD — COMMENTED** [review]
It looks nice (tested)! :)

I only leave a design concern:
- I'm not sure about the Rectangle and flat button: not sure about the hovered color and not sure about the rectangle radius. Could you review this with the designers? 
I guess maybe it is not necessary an hover color and just display the `>` or `<` icons and when user hovers in the area, just change the mouse pointer to hand.

**Comment by MishkaRogachev** [comment]
> @MishkaRogachev could you kindly help with the builds for testing?

Yes, ping me in discord

> * I'm not sure about the Rectangle and flat button: not sure about the hovered color and not sure about the rectangle radius. Could you review this with the designers?
>   I guess maybe it is not necessary an hover color and just display the `>` or `<` icons and when user hovers in the area, just change the mouse pointer to hand.

Yes, i'll change buttons to mouse area and add icons from design

File: ui/app/AppLayouts/CommunitiesPortal/controls/CommunityTagsRow.qml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+import utils 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Controls 0.1
+
+Item {
```
[the remark this answers is not in the document]
Comment (MishkaRogachev) [reply ui/app/AppLayouts/CommunitiesPortal/controls/CommunityTagsRow.qml:12]: It's possible but arrow buttons are outside this scroll area so i would prefer to group them with additional Item

**Review by caybro — APPROVED** [review]
Looks good modulo what Noelia said.

Would look better with rounded/ghost icons

**Comment by MishkaRogachev** [comment]
@noeliaSD fixed and switched to SVG icons

**Review by noeliaSD — APPROVED** [review]

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**